doc/classes/XRCamera3D.xml

```diff
@@ -4,7 +4,7 @@
 		A camera node with a few overrules for AR/VR applied, such as location tracking.
 	</brief_description>
 	<description>
-		This is a helper spatial node for our camera; note that, if stereoscopic rendering is applicable (VR-HMD), most of the camera properties are ignored, as the HMD information overrides them. The only properties that can be trusted are the near and far planes.
+		This is a helper 3D node for our camera; note that, if stereoscopic rendering is applicable (VR-HMD), most of the camera properties are ignored, as the HMD information overrides them. The only properties that can be trusted are the near and far planes.
```

Comment:
While at it:
```suggestion
		This is a helper 3D node for our camera. Note that if stereoscopic rendering is applicable (VR-HMD), most of the camera properties are ignored, as the HMD information overrides them. The only properties that can be trusted are the near and far planes.
```

Reply:
There's two commas that enclose `if stereoscopic rendering is applicable (VR-HMD)` which makes sense, hence I'm keeping that.